crates/cxx-qt-lib/src/qml/qqmlapplicationengine.rs

```diff
@@ -12,6 +12,8 @@ mod ffi {
         type QStringList = crate::QStringList;
         include!("cxx-qt-lib/qurl.h");
         type QUrl = crate::QUrl;
+        include!("cxx-qt-lib/qanystringview.h");
+        type QAnyStringView<'a> = crate::QAnyStringView<'a>;
```

Comment:
you need the cfg flags also here, although unsure if they work on include macros?

As this is a `cxx::bridge` rather than a `cxx_qt::bridge` I think the cfg flag should work on an `extern block`. It's only CXX-Qt bridges where we have the limitation of support only on certain items #1133

Reply:
I tried a ton of solutions, and it doesn't seem like extern blocks support cfg macros. It would get rid of the include, but still try to parse the `type`. I have looked into substituting a dummy type which kinda works, but it's working around missing functionality that should really be in CXX..